firmware/pattern.c

```diff
@@ -12,7 +12,7 @@
 #include "uptime.h"
 #include "version_data.h"
 
-#define PATTERN_FRAMEBUFFER_BASE 0x02000000 + 0x100000
```

Comment:
It would be good if the `0x100000` part was a define somewhere. It's the rough size of the frame buffer right?

Reply:
Who knows. I haven't seen a memory map.

Presumably the framebuffers are stored above this base, so I don't know what goes between 0x02000000 and the base.